### PR TITLE
Use more compact font

### PR DESCRIPTION
**Pull request by rosano**
This is aesthetic but also a bit more compact as the letters are less
wide.

### CODE REVIEW

**Comment by Simon-Laux** [comment]
no visible changes on firefox on linux

**Comment by rosano** [comment]
Before:

Screen Shot 2022-12-15 at 19 15 11

After:
Screen Shot 2022-12-15 at 19 15 09

The first point about messaging anyone fits on one line.

**Comment by rosano** [comment]
> no visible changes on firefox on linux

The fonts might not be available? But nothing was removed, only an extra option added if available.

**Comment by r10s** [comment]
there is still some letter spacing i added that time for h1...h3 - is this still on purpose?

apart from that, indeed, "Lucida Grande" seems not to be considered a web-safe font (random link: https://www.cssfontstack.com) "Lucida Grande" is mostly mac - not even windows.

**Comment by rosano** [comment]
> there is still some letter spacing i added that time for h1...h3 - is this still on purpose?

Wasn't aware of it but could be left in or taken out. I think it's too compressed without the spacing, but with Verdana it's too wide.

> "Lucida Grande" seems not to be considered a web-safe font (random link: https://www.cssfontstack.com) "Lucida Grande" is mostly mac - not even windows.

Updated to use Arial everywhere and Tahoma for headings.

**Comment by missytake** [comment]
On my xubuntu with Firefox this change seems to make it worse right now.

Status quo:

Screenshot_2022-12-16_10-10-35

With this PR:

Screenshot_2022-12-16_10-10-42

**edit:** okay, not necessarily worse, aesthetics are subjective I guess - is this what Tahoma is supposed to look like?

**Comment by rosano** [comment]
> is this what Tahoma is supposed to look like?

Weird because I only see `sans-serif` fonts in the whole project. Can you let me know what's the computed style/font for you? I can only imagine this happening because of browser defaults or custom user CSS. Here's what I see:

Screen Shot 2022-12-16 at 12 36 23

**Comment by missytake** [comment]
I think it's because I don't have Tahoma and there is no fallback font configured?

Screenshot_2022-12-16_12-58-36

**Comment by rosano** [comment]
> I think it's because I don't have Tahoma and there is no fallback font configured?

I thought it would use the body font… @missytake Would you mind editing either in the browser or in source to see if `font-family: Tahoma, inherit;` works? Just because the issue is on your machine. Worst case, we can use `font-family: Tahoma, sans-serif;`

**Comment by missytake** [comment]
`font-family: Tahoma, inherit;` doesn't work, but `font-family: Tahoma, sans-serif;` works :)

**Comment by rosano** [comment]
Ok hopefully that should be resolved for @missytake. Any other feedback?

**Review by missytake — APPROVED** [review]
works now :)

**Comment by r10s** [comment]
the body text is harder to read with this pr.

this is just because the font is overall smaller now - even if the pt size is not changed. is this on purpose?

to get the effective same font-size, it needs to be increased from 15px to 16px and the line-height can be decreased from 1.7em to 1.6em - side by side we would have the same font-size but a bit more compact.

overall, i am +/-0 with this pr,  maybe it is good to have sth. fresh after these years.

**Comment by rosano** [comment]
> needs to be increased from 15px to 16px and the line-height can be decreased from 1.7em to 1.6em - side by side we would have the same font-size but a bit more compact.

Done. I increased the heading size to give more contrast to the body text.

> overall, i am +/-0 with this pr, maybe it is good to have sth. fresh after these years.

I have (perhaps incorrectly) perceived various desires to improve the site design and I'm trying to do small incremental changes rather than big ones, so it might be a more noticeable improvement when they're combined. Perhaps I should also just propose non-incremental changes so that the difference is more obvious.

**Comment by r10s** [comment]
> and I'm trying to do small incremental changes rather than big ones

this is very welcome!

and i think, after some forth and back, font is nice now. different, but nice :)

> so it might be a more noticeable improvement when they're combined. 

good point.

**Review by r10s — APPROVED** [review]

File: _sass/_base.scss [hunk 1]
```diff
@@ -155,17 +155,21 @@ p.blogdate {
 /*colors*/
 
 body {
-    font-family: Verdana, Arial, Helvetica, sans-serif;
-    font-size: 15px;
-    line-height: 1.7em;
+    font-family: Arial, Verdana, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.6em;
```
Comment (r10s) [inline _sass/_base.scss]:
```suggestion
    line-height: 1.5em;
```

did i say 1.6em? triying it over, i mean 1.5em :) with that, and the increased font-size, height and spacing are same as before (and aesthetic :), with, of course, the more compact font.

**Review by r10s — APPROVED** [review]

**Pull request merged**